medicaltorch/transforms.py

```diff
@@ -46,21 +48,25 @@ def __call__(self, sample):
         rdict = {}
         input_data = sample['input']
 
-        if isinstance(input_data, list):
-            ret_input = [F.to_tensor(item)
-                         for item in input_data]
+        if len(input_data) > 1:
+            # Multiple inputs
+            ret_input = [F.to_tensor(item) for item in input_data]
         else:
-            ret_input = F.to_tensor(input_data)
+            # single input
+            ret_input = F.to_tensor(input_data[0])
```

Comment:
ret_input is either a list either a tensor --> does this cause no issue further in the code?

Reply:
`ToTensor()` returns:
- a **tensor**  in case of single input
- a **list of tensors** otherwise.

All transformation (from **MedicalTorch** or **IvadoMed**) that come after the later, deal with both. As **MedicalToch** is used by other people, I've preferred to return the same type as the current master in case of a single input.